### ISSUE TITLE
Icon padding and margin too large with scale factor > 1

### ISSUE THREAD

**Issue by bearoso** [home-sweet-gnome/dash-to-panel#595]
With scale factor > 1, increasing the icon padding shifts the icon off the panel. Looks like the margin is too high as well. Changing lines 531 and 532 of appIcons.js to divide by scaleFactor seems to fix this.

*edit* Values at 1520 need to be scaled, too.

**Comment by charlesg99** [comment]
Hey bearoso, thank you for the report! I can't reproduce this currently, can you tell us your dash-to-panel and gnome-shell versions? We shouldn't have to apply the scale factor on those values since they're applied via CSS and St.Widget (normally) does the appropriate size conversions. Thanks!

**Comment by bearoso** [comment]
Using the latest dash-to-panel git and gnome-shell (3.32) git. 
Here's the panel with no padding:
![nopadding](https://user-images.githubusercontent.com/16159164/54093337-e2a89380-4364-11e9-886d-d31835bba5d0.png)

And here with padding 12 to show how the icons aren't centered:
![padding12](https://user-images.githubusercontent.com/16159164/54093340-e63c1a80-4364-11e9-8a4d-71b9699ea479.png)

And showing the indicators getting pushed off the bottom:
![topoffset](https://user-images.githubusercontent.com/16159164/54093435-cfe28e80-4365-11e9-891c-df18f9948e34.png)

You're right about the CSS properties being correct. As you can see in the third image, the icon is 24 pixels from the top, which at scale factor 2 is right. I think the actual problem is that the icons are only being shrunk by 1/scaleFactor * the pixels they're supposed to. A 32-pixel panel with padding of 12 should mean an icon height of 8, or 16 pixels at scale factor 2, while here they're 40px.

**Comment by bearoso** [comment]
Changing taskbar.js:748 to 
```
let availSize = (this.panelWrapper.panel.actor.get_height() -
                (this._dtpSettings.get_int('appicon-padding') * 2 * scaleFactor)) / scaleFactor;
```
Looks to be the correct fix.

**Comment by charlesg99** [comment]
Ha yes thank you I could reproduce that! On 3.32, gnome-shell changed the BaseIcon space allocation, that explains why I couldn't reproduce the issue on 3.30. Your solution works fine on both 3.30 and 3.32, awesome! Do you want to make a pull request or would you rather we put it in? Thanks!

**Comment by bearoso** [comment]
You can go ahead and put it in.

**Comment by charlesg99** [comment]
Your fix was added to master, thanks again!

**Comment by leebickmtu** [comment]
When will there be a release with this fix? I'm having the same issue on GNOME 3.32 and Dash to Panel v19. I prefer to use official releases from GNOME extensions website not the dev git.
